test/doc/longhaul.md

```diff
@@ -0,0 +1,46 @@
+# Longhaul Tests
+
+## Test Scenarios
+Longhaul tests have the goal of successfully operating all the edge primitives at once. These are test scenarios to be covered in Longhaul test
+
+1. Messaging Test
+   1. Module-to-Module
+   2. Module-to-IoT Hub
+2. Twin Property Update Test
+   1. Desired property
+   2. Reported property
+3. Direct Method Test
+   1. Cloud-to-module
+
+Below I have diagramed some of the specific scenarios.
+
+## General Scenario
+The general scenario is that one deploys an edge instance with the TestResultCoordinator (TRC) and some amount of dummy modules. The dummy modules will take some action and report to the TRC that it happened. The TRC then uses all the context from the reports to generate a success or failure from that test run.
```

Comment:
I think of a "dummy module" as a module that doesn't do anything. Is that what you're describing?

Reply:
They are dumb in the sense that the modules are doing things only for testing. It is better to call them test modules I think.

Reply:
Does LonghaulTest_General.png need to change too?

Reply:
I think with the wording it is fine, but I can change it if you want.

Reply:
What I mean is, the markdown file calls them "test modules", but LonghaulTest_General.png still has "dummy module 1" and "dummy module 2". Your call.